docs/pages/server-access/guides/openssh.mdx

```diff
@@ -33,230 +33,27 @@ and running, but in the long run, we would recommend replacing `sshd` with `tele
-spec:
-  allow:
-    rules:
-      - resources:
-          - host_cert
-        verbs:
-          - list
-          - create
-          - read
-          - update
-          - delete
-```
-
-Create the role resource:
-
-```code
-$ tctl create host-certifier.yaml
-# role 'host-certifier' has been created
-```
-
-(!docs/pages/includes/add-role-to-user.mdx role="host-certifier"!)
-
-You will now have the required permissions to export a host key for your `sshd`
-host.
-
-### Issue a host certificate
-
-<Details title="Find your node's UUID">
-
-When you created a `node` resource and if you didn't set the `metadata.name` field earlier,
-the Teleport Auth Service generated a universal unique identifier (UUID) for that node.
-Teleport Proxy Services uses the UUID to differentiate nodes with the same hostname, so
-it must be added to the host certificate. To find your node's UUID, first determine if its hostname is unique:
-
-```code
-$ tctl get node/openssh-node --format text
-```
-
-If only one node is displayed and you have `jq` installed, you can run the
-following command to get your node's UUID:
-
-```code
-$ tctl get node/openssh-node --format=json | jq -r ".[0].metadata.name"
-```
-
-Otherwise, find your node's UUID in the `metadata.name` field of the YAML
-output of this command:
-
-```code
-$ tctl get node/openssh-node
-```
-
-</Details>
-
-#### Create the host certificate
-
-When creating host certificates, it is important to specify all the domain names
-and addresses that refer to your node. If you try to connect to a node with a 
-name or address that was not specified when creating it's host certificate,
-Teleport will reject the SSH connection.
-
-On your local machine, assign the IP address, fully qualified domain name of
-your node, and the node's UUID to an environment variable. If you won't be
-connecting to your node with its hostname, you can safely omit it.
-
-```code
-$ ADDR=1.2.3.4,openssh-node,a100fdd0-52db-4eca-a7ab-c3afa7a1564a
-```
-
-Run the following `tctl` command to generate a host certificate:
+On the machine the cluster's Proxy is running on, use the `tctl` tool to generate a new token. In the
+following example, a new token is created with a TTL of five minutes:
 
 ```code
-$ tctl auth sign \
-      --host=${ADDR?} \
-      --format=openssh \
-      --out=myhost
-
-# The credentials have been written to myhost, myhost-cert.pub
+$ tctl tokens add --ttl=5m --type=node --format=text
+(=presets.tokens.first=)
 ```
 
-The above command will result in a private key and certificate.
-
-<Details title="Issuing certificates for multiple hosts">
-
-To generate certificates for multiple hosts, assign the `host` flag to a
-comma-separated list of addresses. Certificates for wildcard domains are not
-supported by OpenSSH, so each domain must be fully qualified.
-
-</Details>
-
-Use `ssh-keygen` to verify the contents of the certificate:
+Copy the `teleport` binary to your `sshd` host. Configure `sshd` and create a node
```

Comment:
The introductory section of this page implies that the OpenSSH approach is an alternative to `teleport`:

> Using Teleport and OpenSSH has the advantage of getting you up and running, but in the long run, we would recommend replacing `sshd` with `teleport`. `teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:

However, if we're copying the `teleport` binary to the `sshd` host, what's the alternative presented by this approach? 

I think it would be good to describe in the introduction of this page:
- How OpenSSH support works
- How this is an alternative to the "regular" way of enrolling an SSH server with Teleport